app/utils.js

```diff
@@ -78,3 +78,15 @@ module.exports.downloadAndUnzipFile = function(url, outputDir){
       });
   });
 };
+
+exports.array2String = function (array) {
+  return array.reduce((acc, word) => acc + ' ' + word, '').slice(1);
+};
```

Comment:
should be replace with `array.join()` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join